### PR TITLE
docs: inform about android target sdk 33 requirements

### PR DESCRIPTION
**Pull request by santhoshvai**
Preview from docs

Screenshot

### CODE REVIEW

**Review by khushal87 — APPROVED** [review]

File: packages/react-native-sdk/docusaurus/docs/reactnative/01-setup/02-installation/01-react-native.mdx [hunk 1]
```diff
@@ -39,6 +39,28 @@ So what did we install precisely?
...
+        ...
+        minSdkVersion = 24
+        compileSdkVersion = 33
+        targetSdkVersion = 33
+    }
+    ...
+}
+```
+
+:::infoINFO
```
Comment (khushal87) [inline packages/react-native-sdk/docusaurus/docs/reactnative/01-setup/02-installation/01-react-native.mdx:59]: This should be info right?

File: packages/react-native-sdk/docusaurus/docs/reactnative/01-setup/02-installation/02-expo.mdx [hunk 2]
```diff
@@ -34,6 +34,38 @@ So what did we install precisely?
...
+          "targetSdkVersion": 33
+        }
+      }
+      // highlight-end
+    ]
+  }
+}
+```
+
+:::infoINFO
```
Comment (khushal87) [inline packages/react-native-sdk/docusaurus/docs/reactnative/01-setup/02-installation/02-expo.mdx:64]: Same here

File: packages/react-native-sdk/docusaurus/docs/reactnative/03-core/07-keeping-call-alive.mdx [hunk 3]
```diff
@@ -22,6 +22,28 @@ In Android, we use a [foreground service](https://developer.android.com/guide/co
...
+        ...
+        minSdkVersion = 24
+        compileSdkVersion = 33
+        targetSdkVersion = 33
+    }
+    ...
+}
+```
+
+:::infoINFO
```
Comment (khushal87) [inline packages/react-native-sdk/docusaurus/docs/reactnative/03-core/07-keeping-call-alive.mdx:42]: Same here

File: packages/react-native-sdk/docusaurus/docs/reactnative/01-setup/02-installation/01-react-native.mdx (hunk 1 above)
Comment (santhoshvai) [reply packages/react-native-sdk/docusaurus/docs/reactnative/01-setup/02-installation/01-react-native.mdx:59]: current info shows as success.. so overriding with INFO

Comment (santhoshvai) [reply packages/react-native-sdk/docusaurus/docs/reactnative/01-setup/02-installation/01-react-native.mdx:59]: success is not a good thing to show... 

:::note is not sufficient for an important thing actually

so taking the overriding route

Comment (santhoshvai) [reply packages/react-native-sdk/docusaurus/docs/reactnative/01-setup/02-installation/01-react-native.mdx:59]: you can see in the screenshot how it looks

**Comment by efstathiosntonas** [comment]
@santhoshvai is there any progress in this matter? I've sent a support ticket but I got no information back other than "we will get back to you".

**Pull request merged**